### PR TITLE
PIM-5364: How to avoid rules application on mass edit + style fix & orga

### PR DESCRIPTION
**Pull request by grena**

### CODE REVIEW

File: cookbook/mass_edit.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+How to Customize Mass Edit Operations
+=====================================
+
+This chapter gives details on how to customize existing mass edit operations & how to add some new.
```
Comment (MarieMinasyan) [inline cookbook/mass_edit.rst]: "new ones" :)

File: cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  This cookbook is about a feature only provided in the **Enterprise Edition**.
+
+This cookbook assumes that you already created a new bundle to add your custom rule. Let's assume its namespace is `Acme\\CustomBundle`.
+
+One Service to Override
+-----------------------
+
+| For this cookbook let say we **don't want to apply rules** after a Mass Edit Common Attributes operation.
+| Currently, this operation is defined as it:
```
Comment (MarieMinasyan) [inline cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst]: as follows:

File: cookbook/mass_edit.rst (hunk 1 above)
Comment (SamirBoulil) [inline cookbook/mass_edit.rst]: & -> and

**Comment by MarieMinasyan** [comment]
GTM once fixed :+1:

File: cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst [hunk 3]
```diff
@@ -0,0 +1,113 @@
+How to Avoid Rules Execution on Mass Edit Actions
+=================================================
+
+| The Akeneo PIM comes with a number of mass edit actions.
+| By default in Enterprise Edition, after each of those actions, **rules are recalculated** for edited products.
```
Comment (SamirBoulil) [inline cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst]: **rules are applied** for edited products in enterprise edition by default  -> I find 'applied' more accurate as rules are applied but their operations calculates new values ? ahah :)

File: cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+| If for any reason you don't want rules to apply, then please follow this cookbook.
```
Comment (SamirBoulil) [inline cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst]: "Please follow this cookbook if you don't want rules to apply for some reason"

File: cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst (hunk 2 above)
Comment (SamirBoulil) [inline cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst]: as follow:

File: cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+                - '@pim_user.context.user'
+                - '@pim_catalog.repository.attribute'
+                - '@pim_catalog.updater.product'
+                - '@pim_catalog.validator.product'
+                - '@pim_internal_api_serializer'
+                - '@pim_catalog.localization.localizer.converter'
+                - '@pim_catalog.localization.localizer.registry'
+                - '@pim_enrich.filter.product_values_edit_data'
+                - '%tmp_storage_dir%'
+                - 'edit_common_attributes_with_permission'        # <<<< Notice the different job code (without rules)
```
Comment (SamirBoulil) [inline cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst]: Notice the job code is different (without rules)

File: cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+- ``edit_common_attributes_with_permission_and_rules``: Edit common attributes with EE permission check & rules application (EE Only, EE Default)
+
+**Add to Variant Groups**:
+
+- ``add_to_variant_group``: Edit common attributes (CE Default)
+- ``add_to_variant_group_with_rules``: Edit common attributes (EE Only, EE Default)
+
+.. note::
+
+  All mass edit operation jobs are in minimal data fixtures in ``src/PimEnterprise/Bundle/InstallerBundle/Resources/fixtures/minimal/jobs.yml``
```
Comment (SamirBoulil) [inline cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst]: 'in the minimal data fixtures and declared in'

File: cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst (hunk 3 above)
Comment (SamirBoulil) [inline cookbook/mass_edition/dont_apply_rules_on_mass_edit_action.rst]: for edited products -> or 'for impacted products' -> 'for updated products' you choose ;)

**Comment by SamirBoulil** [comment]
GTM Once fixed ;) good job

**Pull request merged**